helper/pluginutils/hclutils/testing.go

```diff
@@ -40,7 +40,7 @@ func (b *HCLParser) WithVars(vars map[string]cty.Value) *HCLParser {
 // out parameter should be a golang reference to a driver specific TaskConfig reference.
 // The function terminates and reports errors if any is found during conversion.
 //
-// Sample invocation would be
+// # Sample invocation would be
```

Comment:
Maybe this would fix the recommendation? 🤷 

```suggestion
// Sample invocation would be:
```